### DISCUSSION TITLE
Feature Request - Sort Media

### DISCUSSION THREAD

**Discussion by NeuroDawg** [CyferShepard/Jellystat#47] [Ideas]
Now that Jellystat is detecting all my media, it would be nice that when I look at the media, in a library, to see it sorted. Right now it's not clear that there is any type of sort order. The screenshots below shows the media of one of my libraries; it would be nice to have it sorted properly. Right now it looks like some items are sorted by collection title, but then other collections aren't sorted at all. 

![image](https://github.com/CyferShepard/Jellystat/assets/59210675/ac6f327a-129c-41f0-a427-39e5f72c65f9)

![image](https://github.com/CyferShepard/Jellystat/assets/59210675/082f433f-ee35-4abf-984b-8e6757aed441)

**Comment by NeuroDawg** [comment]
@CyferShepard It looks like alphabetization was implemented in 1.0.4.9. It's much appreciated. And now I feel like a nitpiker, but in English the articles 'a' and 'the' are not included when alphabetizing. I would expect to find "A Close Shave" with movies that start with C; "A Separation" with those starting with S; "The Avengers" with those starting with A; and so on. 

Also, would it be possible for Jellystat to read and honor the "sort title" rather than the "title" metadata? For example; I've titled a movie "The Temple of Doom", but I sort it with it's original title "Indiana Jones and The Temple of Doom" because I look for it, and all other Indiana Jones movies as starting with I.

Again, thanks for the great program!

**Comment by CyferShepard** [comment]
Hey @NeuroDawg , ill look into changing how it sorts titles that start with A or The etc. I'm also looking into the sort title. At first glance I don't see it being one of the options available to be read from Jellyfins api, the closest I found was SortTitle but when testing it doesn't seem the represent the Sort Title column. Will keep looking into that. Il only be able to include all this after a few other updates but. I only managed to gap this one in with some bug fixes.
